pandas/tests/indexing/common.py

```diff
@@ -148,9 +148,12 @@ def get_value(self, f, i, values=False):
         # for a in reversed(i):
         #    v = v.__getitem__(a)
         # return v
-        with catch_warnings(record=True):
-            filterwarnings("ignore", "\\n.ix", FutureWarning)
-            return f.ix[i]
+
+        # TODO: this used to be f.ix[i]; is loc-then-iloc correct here?
+        try:
+            return f.loc[i]
+        except KeyError:
```

Comment:
err, this is not good here. can you separate the cases into ones for .loc and ones for .iloc